lib/db/schema/patch-086-087.sql

```diff
@@ -0,0 +1,62 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('86');
+
+CREATE PROCEDURE `verifyTokensWithMethod_3` (
+  IN `tokenIdArg` BINARY(32),
+  IN `verificationMethodArg` INT,
+  IN `verifiedAtArg` BIGINT(1)
+)
+BEGIN
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
+  BEGIN
+    ROLLBACK;
+    RESIGNAL;
+  END;
+
+    -- Update session verification methods
+    UPDATE `sessionTokens` SET verificationMethod = verificationMethodArg, verifiedAt = verifiedAtArg
```

Comment:
I significantly tweaked this procedure to only update the` verificationMethod` on the session table. All previous logic was pulled into the `db.verifyTokensWithMethod` procedure.